packages/k8s/src/k8s/utils.ts

```diff
@@ -137,6 +137,17 @@ exec ${environmentPrefix} ${entryPoint} ${
   }
 }
 
+export function generateContainerName(image: string): string {
+  const nameWithTag = image.split('/').pop()
+  const name = nameWithTag?.split(':').at(0)
+
+  if (name === undefined || name === '') {
```

Comment:
```suggestion
  if (!name) {
```
Sorry for being a stickler :relaxed: This is the last one

Reply:
No worries 😄 

Done 👍 

Not sure if you can; but feel free to push changes to this branch if you happen to spot anything else!